ESPLoader.js

```diff
@@ -629,13 +640,17 @@ class ESPLoader {
         this.log("Running stub...");
         await this.mem_finish(this.chip.ENTRY);
 
-        const res = await this.transport.read({timeout: 1000, min_data: 6});
-        if (res[0] === 79 && res[1] === 72 && res[2] === 65 && res[3] === 73) {
-            this.log("Stub running...");
-            this.IS_STUB = true;
-            this.FLASH_WRITE_SIZE = 0x4000;
-            return this.chip;
-        } else {
+        try {
+            for (let i=0 ; i<100 ; i++) {
```

Comment:
Same

Reply:
Makes sense. Let's add a comment.
```suggestion
            // Check next 100 packets to see if stub is running
            for (let i=0 ; i<100 ; i++) {
```